server/planning/feeding_services/onclusive_api_service.py

```diff
@@ -74,24 +84,32 @@ def _update(self, provider, update):
                 current_event_response.raise_for_status()
 
                 if current_event_response.status_code == 401:
-                    TOKEN = self.renew_token(URL, provider, session)
+                    TOKEN = self.renew_token(provider, session)
                 content = current_event_response.json()
+                self.parse_data(provider, content)
+
+            end_date = current_date + timedelta(days=int(provider["config"]["days"]))
+            for offset in range(100, ONCLUSIVE_MAX_OFFSET, 10000):
+                print("loop started", offset)
+                between_url = "{}/api/v2/events/between?startDate={}&endDate={}&limit={}".format(
+                    URL, current_date.strftime("%Y%m%d"), end_date.strftime("%Y%m%d"), offset
+                )
+                between_event_response = session.get(url=between_url, headers=headers, timeout=TIMEOUT)
+                between_event_response.raise_for_status()
+                content.append(between_event_response.json())
+
+                if between_event_response.status_code == 401:
+                    TOKEN = self.renew_token(provider, session)
```

Comment:
it should probably also call that api again otherwise this would skip this offset and go to next one